### ISSUE TITLE
Add a script that creates new spreadsheets in common format

### ISSUE THREAD

**Issue by attwad** [outbreakprepared/mvp-notes-and-ideas#29]
Should be run like 

```
./create_sheet --config_file=/path/to/CONFIG --name="SomeLocation"
```

- [x] CONFIG is the same config file used by the sheet_cleaner/main.py script, a new section needs to be added with the new sheet id and name.
- [x] new spreadsheet should contain all necessary columns (and validation once/if we add them)
- [x] new spreadsheet should be shared with relevant folks automatically
- [ ] contributors should be made aware of that new spreadsheet if they are responsible for that area (via a new master spreadsheet, Bernardo and team leaders should have write access to it, contributors read access)

**Comment by attwad** [comment]
As we discussed the number at which we should create a new sheet, I made simple calculations:
5M (max limit num of cells) / 24 (current num columns) = 208K

On my macbook a sheet with 24 columns and 150K rows was still editable without any lag/latency issue, let's create a new sheet when the number of rows for a given sheet is between 150K-200K.

**Comment by tbrewer-healthmap** [comment]
Sounds good, thanks for checking that! 

I'm currently trying make things work with service-account as you suggested, but am having a battle with permissions and such (there's a learning curve here).  Basically, I have the service-account set up, it can create a new sheet from the template, but it needs to give permissions to someone otherwise only it can access it (so I'm almost there). 

That being said I definitely see the advantage of using it =)

**Comment by attwad** [comment]
Great! Looking forward to seeing those changes in soon :)
(reminder to not forget to sync your branch before pushing or send it for review my way, please do not break master ;-) )

**Comment by attwad** [comment]
Fixed by #53
